build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/release/UpdateVersionsTask.java

```diff
@@ -198,4 +223,41 @@ static Optional<CompilationUnit> removeVersionConstant(CompilationUnit versionJa
 
         return Optional.of(versionJava);
     }
+
+    private static final int VERSION_ADDITION = 1000;
+
+    @VisibleForTesting
+    static Optional<CompilationUnit> addVersionId(CompilationUnit java, Version releaseVersion) {
+        TypeDeclaration<?> type = java.getType(0);
+
+        String newVersionField = "RELEASE_" + releaseVersion.toString().replace('.', '_');
```

Comment:
This name doesn't seem appropriate. It _might_ end up being the release version, but other versions might be added before release. Could we instead name it something like "INITIAL_WHATEVER"? The version collapsing can then rename the constant as appropriate for what is actually released.